### PR TITLE
Add NAPALM & LDAP Support

### PR DESCRIPTION
**Pull request by rcmurphy**
This may not be suitable for master, but maybe an ldap or full-featured branch?

Either way, this adds the necessary dependencies for both NAPALM support and LDAP authentication.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +13,8 @@ ENV NETBOX_COMMIT 0bb5d229e83def3d2c1c571f1cc5baaabc76c7e0
 RUN mkdir -p /usr/src/netbox \
     && git clone https://github.com/digitalocean/netbox.git /usr/src/netbox \
     && (cd /usr/src/netbox && git checkout -q "$NETBOX_COMMIT") \
-    && (cd /usr/src/netbox && pip install --no-cache-dir -r requirements.txt)
+    && (cd /usr/src/netbox && pip install --no-cache-dir -r requirements.txt) \
+    && (pip install --no-cache-dir django-auth-ldap napalm)
```
Comment (rcmurphy) [inline Dockerfile:17]: This is here instead of above because we don't want to reinstall Django.

**Review by pitkley — CHANGES_REQUESTED** [review]
Hi @rcmurphy, sorry for not responding sooner. Thanks for the PR, the changes look very sensible. Though I have to agree with you: I'm not sure if this would be right for `master`/`develop`.

I have no objections to having these changes on a separate branch though. I have created an `ldap` branch for you to point your PR at. Once the PR is merged, I'll setup Docker Hub to build a tag `ldap` off of this new branch.

Does this sound good to you?

**Comment by rcmurphy** [comment]
@pitkley sounds great! I changed the base as requested; thanks for being willing to do that!

**Review by pitkley — APPROVED** [review]

**Comment by pitkley** [comment]
@rcmurphy the Docker Hub tag has been built. :+1: 

I'll try to keep the `ldap` branch up-to-date with `master` whenever I get to updating it. Thanks for the contribution. :slightly_smiling_face:

**Pull request merged**